packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1082,7 +1085,10 @@ export class ContainerRuntime
 		);
 		const opSplitter = new OpSplitter(
 			chunks,
-			this.context.submitBatchFn,
+			this.context.submitBatchFn === undefined
+				? undefined
+				: (batch: IBatchMessage[], refSqn?: number) =>
+						this.opGroupingManager.submitBatch(this.context, batch, refSqn),
```

Comment:
This is probably a question to @andre4i: If we go this path and runtime always sends only one op (i.e. there are no batches), can compresison / chunking be enabled even for older loaders? I.e. what prevents us operating similar whether we deal with old or new loader, given that we will not expose compression metadata (I think) on op itself (after Kian's changes)

Reply:
I see we do expose the compression marker at https://github.com/microsoft/FluidFramework/pull/14512/files#diff-d6dd5b7a4910beea419ad4e2e60191c88283bac177f1d87b9046be1592a763feR3495 and https://github.com/microsoft/FluidFramework/pull/14512/files#diff-d6dd5b7a4910beea419ad4e2e60191c88283bac177f1d87b9046be1592a763feR3523

but yes, if we keep the workaround inside the OpDecompressor in order to 'detect' compressed messages based on their shape, I don't see why it wouldn't work with even older loaders (<1.4, which don't have submitBatchFn)